src/wallet/coinselection.h

```diff
@@ -20,6 +20,14 @@ static constexpr CAmount CHANGE_UPPER{1000000};
 
 /** A UTXO under consideration for use in funding a new transaction. */
 struct COutput {
+private:
+    /** The output's value minus fees required to spend it. Initialized as the output's absolute value. */
```

Comment:
This comment seems to be outdated now. We should be initializing this with the correct value now.

Reply:
It is outdated, I forgot to update this comment.